### PR TITLE
fix: cross-platform wing cli shebang

### LINKED ISSUE #1157: Shebang in Wing CLI is not portable
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### I tried this:

See https://github.com/winglang/wing/issues/1000#issuecomment-1382618051

https://github.com/winglang/wing/blob/8e4f7b816b23d8a0313297b7829a655d3255827d/apps/wing/bin/wing#L1

`-S` is not portable on all distros. Generally speaking, multiple args in shebang is not portable.

### I expected this:

The shebanged file runs with the given node arguments

### Instead, this happened:

`wing: exec: line 11: illegal option -S`

### Component:

Other

### Wing Version:

_No response_

### Node.js Version:

_No response_

### Platform(s):

Linux

### Anything else?

Alpine Linux under WSL

### PR DESCRIPTION
**Pull request by MarkMcCulloh**
This simplified shebang should be able to run anywhere. The downside is the now-required `spawn`, but I think that's worth revisiting later.

Note: this PR **does not** confirm windows support (although windows may work now 😁), but it does resolve #1157

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Comment by Chriscbr** [comment]
> The downside is the now-required spawn

Where is this? Maybe there is another file that was supposed to be committed?

**Review by Chriscbr — APPROVED** [review]

File: apps/wing/bin/wing [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S node --experimental-wasi-unstable-preview1 --no-warnings
```
Comment (eladb) [inline apps/wing/bin/wing:1]: Just for my understanding, it seems like we can simply split these into two scripts. An entrypoint that runs with regular node and the child that's executed with wasm support. Right?

Feels like there is no real additional value to making this a reentrant script.

Comment (MarkMcCulloh) [reply apps/wing/bin/wing:1]: It is technically possible that wing is being run with node that has the flag enabled already. In that case, the logic in this file does a have a use. I also don't think it's worse than separating, but we could do benchmarking to see if there's a perf difference.

Comment (eladb) [reply apps/wing/bin/wing:1]: 🤷🏻‍♀️

**Pull request merged**